drivers/processor/processr/CMakeLists.txt

```diff
@@ -10,3 +10,4 @@ set_module_type(processr kernelmodedriver)
 add_importlibs(processr hal ntoskrnl)
 add_pch(processr processr.h SOURCE)
 add_cd_file(TARGET processr DESTINATION reactos/system32/drivers FOR all)
+add_driver_inf(processr processr.inf)
```

Comment:
The standard name of the inf file stored within %SystemRoot%/inf is "cpu.inf" on Windows.

Reply:
This applies to a bunch of the others too. There's probably value in keeping the same inf names as Windows, though I couldn't say for sure.

Reply:
I tried, but can't find what can be the reason to keep inf file names the same. Can you help? :)

Reply:
A 3rd-party driver's INF file could "import" an existing Windows INF file via the "LayoutFile" directive to base itself upon.

Reply:
So what do you suggest regarding `usb.inf` and friends - leave them as it is in `media/inf` folder?